src/Transport/Config/ExtensionPoints/AzureServiceBusSanitizationExtensionPoint.cs

```diff
@@ -0,0 +1,25 @@
+namespace NServiceBus
+{
+    using System;
+    using AzureServiceBus;
+    using AzureServiceBus.Addressing;
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    public class AzureServiceBusSanitizationExtensionPoint<T> : ExposeSettings where T : ISanitizationStrategy
+    {
+        SettingsHolder settings;
+
+        public AzureServiceBusSanitizationExtensionPoint(SettingsHolder settings) : base(settings)
+        {
+            this.settings = settings;
+        }
+
+        public AzureServiceBusSanitizationExtensionPoint<T> Hash(Func<string, string> hash)
```

Comment:
Hashing, as sanitization rule, are only applicable when using a strategy other than the default (`ThrowOnFailedValidation`). As per discussion, we'll move all rules for validation & sanitization along with hasing rule under `AzureServiceBusSanitizationExtensionPoint<T>`.